rust/feature-flags/src/flags/flag_service.rs

```diff
@@ -57,7 +57,7 @@ impl FlagService {
                         (Ok(token), false)
                     }
                     Err(e) => {
-                        tracing::debug!("Token validation failed for token '{}': {:?}", token, e);
+                        tracing::error!("Token validation failed for token '{}': {:?}", token, e);
```

Comment:
this is a super useful log that immediately identified the problem.  Let's keep it around.